src/main/scala/lockbook/dev/SettingsUi.scala

```diff
@@ -0,0 +1,66 @@
+package lockbook.dev
+
+import java.util.Optional
+
+import javafx.collections.FXCollections
+import javafx.geometry.Insets
+import javafx.scene.control.ButtonBar.ButtonData
+import javafx.scene.control._
+import javafx.scene.layout.GridPane
+
+class SettingsUi(settingsHelper: SettingsHelper, fileHelper: FileHelper) {
+
+  def getView(): Unit = { //use graphbox
+
+    val dialog: Dialog[Settings] = new Dialog[Settings]()
+
+    dialog.getDialogPane.getButtonTypes.addAll(
+      ButtonType.APPLY,
+      ButtonType.CANCEL
+    )
+
+    val gridPane = new GridPane
+
+    getViewHelper(dialog, gridPane)
+
+    val result: Optional[Settings] = dialog.showAndWait()
+    if (result.isPresent) {
+      SettingsHelper.constructJson(result.get(), fileHelper)
+    }
+  }
+
+  def getViewHelper(dialog: Dialog[Settings], gridPane: GridPane): Unit = {
+    val stylesBox       = new ComboBox[String](FXCollections.observableArrayList(Light.fileName))
```

Comment:
Shouldn't this talk to `SettingsHelper` to figure out what is the current setting?

I think there's 2 concepts here and perhaps you want to make the code reflect that:
1. getting all the possible settings
2. getting the currently selected setting

for 1, perhaps you want to add a `val` to the `Setting` (currently called `DefaultSetting`) file that has a `List[Theme]` 

something like:

```
trait Theme extends Setting {...}
object SettingChoices {
    val allThemes: List[Theme] = List(Light)
}
```

Reducing the number of places we need to go and change everytime someone adds a theme reduces the chance of error.